src/styles/main.scss

```diff
@@ -0,0 +1,120 @@
+$watch-size: 400px;
+$pin-size: $watch-size/30;
+
+@mixin watch-hand($w, $h, $clr, $br-clr, $spin-dur) {
+  position: absolute;
+  bottom: 50%;
+
+  box-sizing: border-box;
+  width: $w;
+  height: $h;
+  border-radius: $w/2;
+  border: 1px solid $br-clr;
+
+  background-color: $clr;
+
+  transform-origin: bottom;
+  animation: spin $spin-dur steps(60, end);
+  animation-play-state: paused;
+  animation-iteration-count: infinite;
+}
+
+@mixin sircle ($d) {
+  width: $d;
+  height: $d;
+  border-radius: $d/2;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  height: 100vh;
+  margin: 0;
+
+  font-family: Courier, monospace;
+  font-size: 14px;
+}
+
+.stop-watch {
+  @include sircle($watch-size);
+
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+
+  border: 2px solid black;
+
+  &__reset {
+    position: absolute;
+    left: 0;
+    bottom: 0;
+
+    padding: 0;
+    border: none;
+
+    font-family: inherit;
+    font-size: inherit;
+
+    background: none;
+    cursor: pointer;
+  }
+
+  &__checkbox-label {
+    &::before {
+      position: absolute;
+      right: 0;
+      bottom: 0;
+
+      content: "> start";
+      cursor: pointer;
+
+      .stop-watch__start-stop:checked ~ & {
```

Comment:
Use `scss` nesting

Reply:
It's already nested 3 times (right part) & it's more clear that content of ::before is changing